### PR TITLE
Doc updates 24q1

### LINKED ISSUE #1080: Minor doc/tutorial additions.
Just noticed we did not entirely update the docs/tutorials for some additions at the last release (`Coef` and SDP for example).

https://computationalcryoem.github.io/ASPIRE-Python/auto_tutorials/aspire_introduction.html#pipeline-roadmap

I'll try to skim through docs and make a few touch ups.

### PR DESCRIPTION
**Pull request by garrettwrong**
Resolves #1080 .  These are the entries noted in the issue. WIP, still plan to review gallery for other updates...

### CODE REVIEW

File: gallery/tutorials/aspire_introduction.py [hunk 1]
```diff
@@ -660,7 +662,7 @@ def noise_function(x, y):
 # +----------------+--------------------+-----------------+----------------+---------------------+
 # |CoordinateSource| whiten             |                 | CLSymmetryCn   |                     |
 # +----------------+--------------------+-----------------+----------------+---------------------+
-# |                | phase_flip         |                 |                |                     |
+# |                | phase_flip         |                 | CommonlineSDP  |                     |
```
Comment (j-c-c) [inline gallery/tutorials/aspire_introduction.py]: Can add `CLSymmetryC2`.

**Review by j-c-c — CHANGES_REQUESTED** [review]
Looks good. Just one addition.

File: gallery/tutorials/aspire_introduction.py (hunk 1 above)
Comment (garrettwrong) [reply gallery/tutorials/aspire_introduction.py]: Thanks! I forgot about that one.  Haven't tried it yet :).

Added

**Comment by garrettwrong** [comment]
@j-c-c , while I was in there adding C2, I made some small changes to some `show` calls also. I think they look a little better now, but lmk, thanks!

**Review by j-c-c — APPROVED** [review]
Looks good. Plots look nice!

**Review by janden — APPROVED** [review]
:+1:

**Pull request merged**